src/store/index.ts

```diff
@@ -1,8 +1,15 @@
 import { createStore } from "vuex";
+import { QuestionBucketingMap } from "../types"
 
 export default createStore({
   state: {
     isSpinnerShown: false,
+    questionsBucketingMap: {} as QuestionBucketingMap,
```

Comment:
somewhere it is questionBucketingMap and somewhere question`s`BucketingMap - best to keep it consistent